src/Type/Definition/FieldDefinition.php

```diff
@@ -174,12 +175,9 @@ public function getArg($name)
         return null;
     }
 
-    /**
-     * @return OutputType&Type
```

Comment:
should be kept in order not to lower type coverage

Reply:
As discussed [previously](https://github.com/webonyx/graphql-php/pull/557#discussion_r437488563), nothing I can do here without asserts.